util/src/main/i18n/templates/moderation.properties

```diff
@@ -175,3 +175,5 @@ vanish.chat.deny = You can only use the admin chat while vanished
 
 vanish.hotbar = You are currently vanished
 
+vanish.login.auto = Majority of staff are vanished. Due to this we've vanished you as well
```

Comment:
I don't understand this feature

Reply:
Hey Electroid,
So this idea was suggested by @Pablete1234 yesterday. Basically if more than half of the online staff are vanished, any joining staff will be auto-vanished to prevent players from knowing there are staff online (example for watching a suspected hacker).

Reply:
This is a nice feature, there may be 2 or 3 staff members online who are watching a hacker vanished, and suddently you join unvanished, broadcasting a join-message to all players. This has already happened and can ruin entire operations